app/src/utils/Utils.ts

```diff
@@ -397,3 +397,63 @@ export const getCodesName = (
   }
   return name;
 };
+
+/**
+ * Convert a UUID into an arbitrary color within a constrained color space.
+ *
+ * @param id uuid
+ * @returns {*} {fillColor: string, outlineColor: string}
+ */
+export const uuidToColor = (id: string): string => {
+  const uuidToInt = (uuid: string): number => {
```

Comment:
Could avoid using regex:
```javascript
const intUuid = parseInt(id.split('-')[4]);
```

Reply:
This gives different results which required adjusting the algorithm so not really worth it imo.